.eslintrc.js

```diff
@@ -4,12 +4,7 @@ module.exports = {
     es2021: true,
     jest: true,
   },
-  extends: [
-    'airbnb-base',
-    'plugin:storybook/recommended',
-    'plugin:prettier/recommended',
-    'plugin:@typescript-eslint/recommended',
-  ],
+  extends: ['airbnb-base', 'plugin:storybook/recommended'],
```

Comment:
Why did we go ahead in removing this?

Reply:
there were a lot of errors whih were not being show by eslint. 
1. it wasn't showing error when some files were unused
2. it wasn't showing error if some value ould be `null/indefined`
3. it was not ordering impoerts in a particular order.